cypress/e2e/ui-tests/test-deploy-app.cy.js

```diff
@@ -276,7 +276,7 @@ describe("Test deploying app", () => {
     it("can deploy a dash app", { defaultCommandTimeout: 100000 }, () => {
         // Names of objects to create
         const project_name = "e2e-deploy-app-test"
-        const app_name = "e2e-dash-example"
+        let app_name = "e2e-dash-example"
```

Comment:
I like everything, but if I were to nitpick, I would say that maybe it's better to have two constants and call them `app_name_initial` `app_name_edited`.

I just think that it might make things more explicit, but it's good as it is right now:)

Reply:
Thanks for the comment!